roles/xmpp/tasks/prosody.yml

```diff
@@ -37,6 +34,23 @@
   template: src=prosody.cfg.lua.j2 dest=/etc/prosody/prosody.cfg.lua group=root owner=root
   notify: restart prosody
 
+- name: Copy SSL private key and cert
```

Comment:
Yes, it's a good idea.  The open issue at letsencrypt is exactly right.  I'll go ahead and make the change to fix permissions and the corresponding update to the prosody role to avoid copying the files.